www/src/pages/user/Orders.js

```diff
@@ -53,9 +53,15 @@ class Orders extends React.Component {
                         </Card.Header>
```

Comment:
Header should probably be clickable to go to the real theme instead how having to create another detail view. Checkout some tips from Amazon on how to display purchases.